### DISCUSSION TITLE
Help us Write Docs!

### DISCUSSION THREAD

**Discussion by anjakefala** [saulpw/visidata#2326] [Ideas]
Hello everyone!

We just posted a pair of blogposts about the Documentation Drive of 2024!

This first blogpost introduces Guides: https://www.visidata.org/blog/2024/introducing-guides/, which are the core pieces that we will be writing.

The second blogpost shows where to find our documentation to-do list, and how to participate: https://www.visidata.org/blog/2024/help-with-docs/. 

All doc contributors are viewed as real contributors: after a few substantive PRs, we'll send you some VisiData stickers (including a holographic contributor sticker) and a thank you notecard. We really hope you'll join in! Documentation is a lot better when it is written by folks from the community, who have more recent memory of learning VisiData.

**Comment by almereyda** [comment]
Can you tell us, where one can post a PR to contribute to the website?

**Comment by anjakefala** [comment]
@almereyda Sure!

The website source is hosted here: https://github.com/visidata/visidata.org
